app/aboot/aboot.c

```diff
@@ -4754,7 +4813,7 @@ void cmd_reboot_fastboot(const char *arg, void *data, unsigned sz)
 		return;
 	}
 	fastboot_okay("");
-	reboot_device(REBOOT_MODE_UNKNOWN);
+	reboot_device(RECOVERY_MODE);
```

Comment:
I don't exactly understand how "reboot fastboot" ends up with "recovery mode"

Reply:
Ah, right... So new android needs the fastbootd for the virtual parititons mess I guess. It makes sense I guess though not sure if it'd break the usecase of rebooting into oem bootloader/lk1st fastboot. Need to think about it...